### PR TITLE
feat: translate known addresses

### LINKED ISSUE #451: Show names for known addresses
### Please describe the problem that your request should resolve.
There are "known" addresses that don't have associated AENS names and get shown currently just as addresses. It complicates the investigation of the transaction history.

### Describe the solution you'd like
For example, on aeknow.org
ak_dArxCkAsk1mZB1L9CX3cdz1GDN4hN84L3Q8dMLHN4v8cU85TF showed as 2miners (mining pool?)
ak_6sssiKcg7AywyJkfSdHz52RbDUq5cZe4V4hcvghXnrPz4H4Qg as gate.io

### Is it already possible to achieve the same outcome in any other way? If so, how?
To claim names for these addresses 🙃 but probably they won't do it

### Additional context
https://www.aeknow.org/address/wallet/ak_6sssiKcg7AywyJkfSdHz52RbDUq5cZe4V4hcvghXnrPz4H4Qg

Screenshot 2023-08-16 at 15 39 32

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #451 

known addresses are listed here https://github.com/aeternity/aescan/pull/573/files#diff-bd2e652035adaf33b6125f14a823cf76feae0c1c1a592e9a6c14a14951a52a28R96

## Demo

https://github.com/aeternity/aescan/assets/15363559/372f2133-1736-483c-a6aa-493ae1a3034d

## Checklist:
- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] show translated addresses

### CODE REVIEW

File: src/utils/constants.js [hunk 1]
```diff
@@ -93,3 +93,19 @@ export const REMOVE_LIQUIDITY_CONTRACT_CALLS = [
   'remove_liquidity',
   'remove_liquidity_ae',
 ]
+export const KNOWN_ADDRESSES = [
+  { hash: 'ak_jgiBUko9fTwg5HQ8nHFoPbjR22TBcg9LK85qsv1mTkcjdszFa', name: 'CoinEx' },
+  { hash: 'ak_6sssiKcg7AywyJkfSdHz52RbDUq5cZe4V4hcvghXnrPz4H4Qg', name: 'Gateio' },
```
Comment (michele-franchi) [inline src/utils/constants.js:98]:
```suggestion
  { hash: 'ak_6sssiKcg7AywyJkfSdHz52RbDUq5cZe4V4hcvghXnrPz4H4Qg', name: 'Gate.io' },
```

**Review by michele-franchi — APPROVED** [review]
Looks good, we could also display it in the account detail but it's already a good starting point.

**Comment by janmichek** [comment]
> Looks good, we could also display it in the account detail but it's already a good starting point.

incorporated to account detail. Hope that's how you imagined it .)

File: src/components/AccountDetailsPanel.vue [hunk 2]
```diff
@@ -9,12 +9,13 @@
...
 
       <copy-chip
-        :label="accountDetails.id"
-        class="account-details-panel__copy-chip"/>
+        :label="formatKnownAddress(accountDetails.id, false)"
+        :clipboard-text="accountDetails.id"
+        class="u-hidden-mobile"/>
       <copy-chip
-        :label="formatEllipseHash(accountDetails.id)"
+        :label="formatKnownAddress(accountDetails.id)"
```
Comment (janmichek) [inline src/components/AccountDetailsPanel.vue:16]: I touched this and noticed the usage of styling classes for the different views, which I recently replaced. Here was forgotten, so I adjusted it also in sibling files

**Review by Liubov-crypto — APPROVED** [review]
LGTM

**Review by michele-franchi — CHANGES_REQUESTED** [review]
It looks good but I would not apply it in the account detail like this, I have some other ideas on how to do it and I'll open a followup.
Please remove the `AccountDetailsPanel` implementation and I'll approve.

**Comment by janmichek** [comment]
> It looks good but I would not apply it in the account detail like this, I have some other ideas on how to do it and I'll open a followup. Please remove the `AccountDetailsPanel` implementation and I'll approve.

Removed

**Review by michele-franchi — APPROVED** [review]
Good job

**Pull request merged**